docs/api/topics/firefox_os_addons.rst

```diff
@@ -38,6 +38,7 @@ Detail
         {
           "id": 1,
           "description": null,
+          "disabled": false,
```

Comment:
I could change it to `is_disabled` everywhere, including in the model, if you prefer.